examples/interpret.rs

```diff
@@ -14,23 +14,25 @@ fn main() {
         return;
     }
 
-    // Intrepreter initialization.
-    let program = parity_wasm::ProgramInstance::new();
-
     // Here we load module using dedicated for this purpose
     // `deserialize_file` function (which works only with modules)
     let module = parity_wasm::deserialize_file(&args[1]).expect("Failed to load module");
 
+    let validated_module = validate_module(module).expect("Failed to validate module");
+
     // Intialize deserialized module. It adds module into It expects 3 parameters:
-    // - a name for the module
-    // - a module declaration
-    // - "main" module doesn't import native module(s) this is why we don't need to provide external native modules here
-    // This test shows how to implement native module https://github.com/NikVolf/parity-wasm/blob/master/src/interpreter/tests/basics.rs#L197
-    let module = program.add_module("main", module, None).expect("Failed to initialize module");
+     // - a name for the module
+     // - a module declaration
+     // - "main" module doesn't import native module(s) this is why we don't need to provide external native modules here
+     // This test shows how to implement native module https://github.com/NikVolf/parity-wasm/blob/master/src/interpreter/tests/basics.rs#L197
```

Comment:
should link to a specific commit blob instead of master

Reply:
yeah, agree.
btw this comment is very outdated